.pre-commit-config.yaml

```diff
@@ -19,6 +19,8 @@ repos:
   - repo: https://github.com/astral-sh/ruff-pre-commit
     rev: v0.6.3
     hooks:
+      - id: ruff
+        args: [--fix, --exit-non-zero-on-fix]
```

Comment:
Does exit-non-zero-on-fix correctly commit the fixes?

Reply:
Yes - we want an exit code of zero _only_ if there have been no changes, otherwise the non-zero code prompts pre-commit that things weren't ok and that something in the commit needs changing (even if that's just adding the things which `--fix` has tidied up for us).